composer.json

```diff
@@ -1,11 +1,16 @@
 {
   "name": "spryker-eco/punchout-catalogs",
+  "type": "library",
   "description": "PunchoutCatalogs module",
-  "license": "MIT",
+  "license": "proprietary",
   "require": {
     "php": ">=7.1",
-    "spryker/kernel": "^3.0.0",
-    "punchout-catalogs/punchout-catalog-spryker": "dev-master"
+    "punchout-catalogs/punchout-catalog-spryker": "^1.0.0",
+    "spryker/company-business-unit": "^2.0.0",
+    "spryker/company-user": "^2.0.0",
+    "spryker/kernel": "^3.33.0",
+    "spryker/util-text": "^1.1.0",
+    "spryker/vault": "^1.0.0"
   },
```

Comment:
In the Persistence layer, you nicely put the `@module` tags, but unfort. it seems like the tooling was not alerting you that you need `company` and `customer` modules in the composer